README.md

```diff
@@ -38,66 +38,78 @@ Function        Set-CloudEventJsonData                             0.2.0      Cl
 Function        Set-CloudEventXmlData                              0.2.0      CloudEvents.Sdk
 ```
 
-## Event Producer
+## Using **CloudEvents.Sdk** Module
+## 1. Event Producer
 ### Create a CloudEvent object
 ```powershell
-$cloudEvent = New-CloudEvent -Type 'com.example.object.deleted.v2' -Source 'mailto:cncf-wg-serverless@lists.cncf.io' -Id '6e8bc430-9c3a-11d9-9669-0800200c9a66' -Time (Get-Date)
+$cloudEvent = New-CloudEvent -Type 'com.example.object.deleted.v2' -Source 'mailto:cncf-wg-serverless@lists.cncf.io' -Id (New-Guid).Guid -Time (Get-Date)
 ```
 
-### Add **JSON Data** to a CloudEvent object
+### Set **JSON Data** to a CloudEvent object
 ```powershell
-$cloudEvent | Add-CloudEventJsonData -Data @{
-   'key1' = 'value1'
-   'key2' = @{
-      'key3' = 'value3'
-   }
+$cloudEvent | Set-CloudEventJsonData -Data @{
+    'Foo' = 'Hello'
+    'Bar' = 'World'
 }
 ```
-
-### Add **XML Data** to a CloudEvent object
-```powershell
-$cloudEvent | Add-CloudEventXmlData -Data @{
-   'key1' = @{
-      'key2' = 'value'
-   }
+The cloud event object is populated with the following properties
+```powershell
```

Comment:
Can you please also show the command to print/get this output?

Reply:
Sure, there is no additional command actually, the above one displays it that way. I'll combine both in a single code block.

Reply:
Ah nice!